CorsixTH/Lua/research_department.lua

```diff
@@ -326,79 +326,71 @@ function ResearchDepartment:getResearchRequired(thing)
   return required
 end
 
---[[ Add some more research points to research progress. If
-autopsy_room is specified points are not used. Instead research
-progresses according to the level config for autopsies.
-Otherwise they will be divided according to the research policy
-into the different research areas.
-!param points (integer) The total amount of points (before applying
-any level specific divisors to add to research.
-!param autopsy_room (string) If a specific room should get points following
-an autopsy, then this is the id of that room.
-]]
-function ResearchDepartment:addResearchPoints(points, autopsy_room)
-
+--! This function will add progress to discovering a room/object to treat a diagnosed patient after an autopsy.
```

Comment:
So you kill a patient first, then treat it?
Sounds a bit fishy :p

Perhaps something like "Give research points to the treatment of illness of the autopsied patient."?